content/collections/advanced-techniques/en/flag-prerequisites.md

```diff
@@ -6,50 +6,51 @@ exclude_from_sitemap: false
+Amplitude Experiment lets you create dependencies based on prerequisite flags or experiments.
 
 ### Feature availability
 
-This feature is available to users on **Enterprise plans** who have **purchased Amplitude Experiment**. See the [pricing page](https://amplitude.com/pricing) for more details.
+This feature is available to Enterprise plan users who have purchased Amplitude Experiment. See the [pricing page](https://amplitude.com/pricing) for more details.
 
 ## Configuring flag prerequisites
 
-Configure flag prerequisites in the _Dependencies card_, which you can find below the _Overview card_ for experiments and below the _Settings card_ for flags.
+You can configure flag prerequisites in the Dependencies card, located below the Overview card for experiments and below the Settings card for flags.
 
-For a given flag or experiment, this card provides a summary of all flag dependencies including its prerequisite flags, experiments, mutual exclusion, and holdout groups as well as a list of the flags and experiments that are dependent on it.
+This card provides a summary of all flag dependencies, including prerequisite flags, experiments, mutual exclusions, and holdout groups, as well as a list of flags and experiments dependent on it.
 
-1. To configure new prerequisites, click the edit icon.
-2. In the Dependencies edit drawer, click _Add Dependency_ to add a new prerequisite flag or experiment.
-3. Select the flag or experiment you wish to create the dependency on.
+1. To configure new prerequisites, select the edit icon.
+2. In the Dependencies edit drawer, select _Add Dependency_ to add a new prerequisite flag or experiment.
+3. Select the flag or experiment you want to create the dependency on.
 
 	{{partial:admonition type='note'}}
 	Flags and experiments are eligible to be used as a prerequisite if:
 
 	- They're in the same project.
-	- They have compatible evaluation modes. Local evaluation mode flags and experiments can only have local evaluation mode prerequisites. Remote evaluation mode flags and experiments can have both remote and local prerequisites.
+	- They've compatible evaluation modes. Local evaluation mode flags and experiments can only have local evaluation mode prerequisites, while remote evaluation mode flags and experiments can have both remote and local prerequisites.
 
-	You can't add a prerequisite that would cause a circular dependency loop.
+	You can’t add a prerequisite that would create a circular dependency loop.
 	{{/partial:admonition}}
 
-1. Then, select the variants that you want to be dependent on. _Off_ is a special value that represents when users weren't included in the prerequisite flag or experiment.
-2. To finish adding the dependency, click _Save_.
+1. Then, select the variants that you want to be dependent on. Off is a special value that represents when users weren't included in the prerequisite flag or experiment.
```

Comment:
```suggestion
1. Select the variants that you want to be dependent on.`off` is a special value that represents when users weren't included in the prerequisite flag or experiment.
```

If you haven't, please look along at the UI as you go through these to ensure option names and casing match.